vtm/src/org/oscim/layers/tile/buildings/BuildingLayer.java

```diff
@@ -43,8 +44,8 @@
 
     protected final static int BUILDING_LEVEL_HEIGHT = 280; // cm
 
-    protected final static int MIN_ZOOM = 17;
-    protected final static int MAX_ZOOM = 17;
+    public final static int MIN_ZOOM = 17;
+    public final static int MAX_ZOOM = TileSource.MAX_ZOOM;
```

Comment:
`BuildingLayer` properties better remain self handled.
Tile sources currently depend on buildings **not** overzoom, so need to have max zoom same as buildings if wish to avoid the multiple-render effect.